src/index.css

```diff
@@ -117,4 +117,20 @@ body.dark .react-loading-skeleton{
     var(--backgroundGrey)
   );
 }
-
+.remirror-editor {
+  min-height: 250px !important;
```

Comment:
Why the important here. Is this because remirror does not play nicE?

Reply:
Editor stayed small and emoji list stayed too small to be usable. you would only see like 3 options